Resources/Prototypes/_Echo/Entities/Objects/Fun/centcom_pai.yml

```diff
@@ -0,0 +1,66 @@
+- type: entity
+  parent: PersonalAI
+  id: CentComPersonalAI
+  name: CentCom personal ai device
+  description: Your Central Command pal who might be fun to be with!
+  components:
+  - type: PAI
+    namePrefix: pai-system-name-prefix-centcom
+    paiName: pai-system-name-centcom
+    paiNameRaw: pai-system-name-raw-centcom
+  - type: PAICentCom
+  - type: Sprite
+    sprite: _Echo/Objects/Fun/centcom_pai.rsi
+    layers:
+    - state: centcom-pai-base
+    - state: centcom-pai-off-overlay
+      shader: unshaded
+      map: [ "screen" ]
+  - type: ToggleableGhostRole
+    roleName: pai-system-role-name-centcom
+    roleDescription: pai-system-role-description-centcom
+    roleRules: ghost-role-information-familiar-rules
+  - type: Access
+    groups:
+    - AllAccess
+  - type: Tag
+    tags:
+    - DoorBumpOpener
```

Comment:
I think you misunderstood, when we said "master key" we meant the "station master encryption key".

TL;DR, Remove this, and set the radio channels to:
  - CentCom
  - Common
  - Command
  - Engineering
  - Medical
  - Science
  - Security
  - Service
  - Supply
  - Prison

Reply:
Was already working on making all access an action to prevent abuse and wanton killing of little innocent pAI's. Also, it'll make the pAI player feel usefull.

Reply:
Give them Binary and CentCom as intrinsic. All others are now encryption keys.